ai/ai-endpoints/car-damage-verification-using-vlm/README.md

```diff
@@ -0,0 +1,131 @@
+# VLM Tutorial - Car Damage Verification
+
+This Vision Language Model (VLM) tutorial features an interactive car verification challenge powered by OVHcloud AI Endpoints.
+
+## Files
+
+- `test_vision_connection.py` - Test OVHcloud Vision API connectivity
+- `verification_demo.py` - Core car verification logic using VLM
+- `verification_app.py` - Interactive Chainlit web application
+- `chainlit.md` - Welcome page content for the Chainlit app
+- `requirements.txt` - Python dependencies
+
+## What This Demo Does
+
+The **Car Verification Challenge** is an interactive AI-powered fact-checking experiment where users:
+
+1. **Make claims** about their car (manufacturer, model, color, damage)
+2. **Upload photos** of the actual vehicle (minimum 3 photos)
+3. **Get AI analysis** that verifies if the photos match their claims
+4. **Receive a verdict** - did you tell the truth or try to trick the AI?
+
+This demonstrates how Vision Language Models can analyze visual content and cross-reference it with textual claims for verification tasks.
+
+## Usage
+
+### 1. Prerequisites
+
+Follow the [complete VLM tutorial setup guide](https://cougz.github.io/ovhcloud-workbooks/public-cloud/ai-endpoints/vlm-tutorial-car-damage-verfication/setup-guide/) for detailed setup instructions.
```

Comment:
you can't point to a personal website, this workbook must be hosted on OVHcloud or an official OVHcloud ressource